tests/reflection_test.h

```diff
@@ -1,6 +1,7 @@
 #ifndef TESTS_REFLECTION_TEST_H
 #define TESTS_REFLECTION_TEST_H
 
+#include <stdint.h>
```

Comment:
This type is defined in the C header <stdint.h> which is part of the C++ standard sice 2011, is it necessary to import it?

Reply:
Before C++11 you could not reliably include it as it was not a standard header. From C++11 on you can and should include it if you want to pull `uint8_t` into scope: `stdint.h` for global scope, `<cstdint>` for `std::` scope. Standard defines the header as https://en.cppreference.com/w/cpp/types/integer

It's still not a fundamental type like `int` or `wchar_t`. Those would not require any headers. Standard defines list of fundamental types as https://en.cppreference.com/w/cpp/language/types